fineract-provider/src/main/java/org/apache/fineract/infrastructure/gcm/domain/DeviceRegistration.java

```diff
@@ -42,40 +49,8 @@ public final class DeviceRegistration extends AbstractPersistableCustom {
     @Column(name = "updatedon_date", nullable = false)
     private LocalDateTime updatedOnDate;
 
-    private DeviceRegistration() {}
-
-    private DeviceRegistration(final Client client, final String registrationId) {
-        this.client = client;
-        this.registrationId = registrationId;
-        this.updatedOnDate = DateUtils.getLocalDateTimeOfTenant();
-    }
-
     public static DeviceRegistration instance(final Client client, final String registrationId) {
-        return new DeviceRegistration(client, registrationId);
-    }
-
-    public Client getClient() {
-        return this.client;
-    }
-
-    public void setClient(Client client) {
-        this.client = client;
-    }
-
-    public String getRegistrationId() {
-        return this.registrationId;
-    }
-
-    public void setRegistrationId(String registrationId) {
-        this.registrationId = registrationId;
-    }
-
-    public LocalDateTime getUpdatedOnDate() {
-        return this.updatedOnDate;
-    }
-
-    public void setUpdatedOnDate(LocalDateTime updatedOnDate) {
-        this.updatedOnDate = updatedOnDate;
+        return new DeviceRegistration().setRegistrationId(registrationId).setClient(client);
```

Comment:
`this.updatedOnDate = DateUtils.getLocalDateTimeOfTenant(); ` -> this is missing according to the original constructor.

Reply:
done